config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.2
+    controller-gen.kubebuilder.io/version: v0.16.3
```

Comment:
suggested to commit also edge connect CRD
```
dynatrace-operator git:(refactor/crd_rename_prometheus_flag) ✗ gs
On branch refactor/crd_rename_prometheus_flag
Your branch is up to date with 'origin/refactor/crd_rename_prometheus_flag'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   config/crd/bases/dynatrace.com_edgeconnects.yaml
```

Reply:
fixed in d9092afdaf3ecc66dc0ec59f1b9ef2717173a97c